index.d.ts

```diff
@@ -0,0 +1,28 @@
+declare module "react-native-matomo" {
+    namespace BNFMatomo {
+        export function initTracker(apiUrl: string, siteId: number);
```

Comment:
Nice!. Maybe it would be good idea to add return types also (they my all return void or Promise<void>)

Reply:
I did at the beginning but got a complaint from app compiler
if you find a solution it is very welcome